src/OctoshiftCLI.Tests/gei/Commands/GenerateReclaimCSVCommandTests.cs

```diff
@@ -0,0 +1,174 @@
+using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using OctoshiftCLI.GithubEnterpriseImporter;
+using OctoshiftCLI.GithubEnterpriseImporter.Commands;
+using OctoshiftCLI.Models;
+using Xunit;
+
+namespace OctoshiftCLI.Tests.GithubEnterpriseImporter.Commands
+{
+    public class GenerateReclaimCSVCommandTests
+    {
+        private const string TARGET_API_URL = "https://api.github.com";
```

Comment:
no need to set this since it is the default API url.

Reply:
done